account_financial_report/tests/test_general_ledger.py

```diff
@@ -493,3 +494,57 @@ def test_04_unaffected_account_balance_2_years(self):
         self.assertEqual(lines['unaffected'].final_debit, 500)
         self.assertEqual(lines['unaffected'].final_credit, 0)
         self.assertEqual(lines['unaffected'].final_balance, 500)
+
+    def test_partner_filter(self):
+        partner_1 = self.env.ref('base.res_partner_1')
+        partner_2 = self.env.ref('base.res_partner_2')
+        partner_3 = self.env.ref('base.res_partner_3')
+        partner_4 = self.env.ref('base.res_partner_4')
+        partner_1.write({'is_company': False,
+                         'parent_id': partner_2.id})
+        partner_3.write({'is_company': False})
+
+        expected_list = [partner_2.id, partner_3.id, partner_4.id]
+        context = {
+            'active_ids': [
+                partner_1.id, partner_2.id, partner_3.id, partner_4.id
+                ],
+            'active_model': 'res.partner'
+            }
+
+        wizard = self.env["general.ledger.report.wizard"].with_context(context)
+        self.assertEqual(wizard._default_partners(), expected_list)
+
+    def test_validate_date(self):
+        company_id = self.env.ref('base.main_company')
+        company_id.write({
+            'fiscalyear_last_day': 31,
+            'fiscalyear_last_month': 12,
+        })
+        cr = self.registry.cursor()
+        user = self.env.ref('base.user_root').with_context({
+            'company_id': company_id.id})
+        env = Environment(cr, user.id, {})
```

Comment:
Any good reason to declare this env using the same cursor ?

Reply:
leftover from 10 version, thanks,
 updated